endpoints/openrtb2/video_auction.go

```diff
@@ -298,7 +298,7 @@ func (deps *endpointDeps) createImpressions(videoReq *openrtb_ext.BidRequestVide
 
 		impsArray := make([]openrtb.Imp, numImps)
 		for impInd := range impsArray {
-			newImp := createImpressionTemplate(storedImp, videoData)
+			newImp := createImpressionTemplate(storedImp, *videoData)
```

Comment:
Would it be better just pass down the pointer?

Reply:
Updated to use a pointer argument